src/InputOutputRealm.C

```diff
@@ -64,15 +65,45 @@ InputOutputRealm::initialize_prolog()
 {
   // bar minimum to register fields and to extract from possible mesh file
   register_io_fields();
+  if (solutionOptions_->meshMotion_)
+    register_mesh_motion_fields();
+
   ioBroker_->populate_mesh();
   ioBroker_->populate_field_data();
   create_output_mesh();
   input_variables_from_mesh();
+  if (solutionOptions_->meshMotion_) {
+    init_current_coordinates();
+    meshMotionAlg_->initialize(get_current_time());
```

Comment:
Seg faulting here inside the `get_current_time()` call. End of the stack trace is inside the `timeIntegrator_` when it should be returning the `double` for time.